lib/inline_svg/webpack_asset_finder.rb

```diff
@@ -6,32 +6,45 @@ def self.find_asset(filename)
 
     def initialize(filename)
       @filename = filename
+      @asset_path = Webpacker.manifest.lookup(@filename)
     end
 
     def pathname
-      file_path = Webpacker.instance.manifest.lookup(@filename)
-      return unless file_path
+      return if @asset_path.blank?
 
       if Webpacker.dev_server.running?
-        asset = Net::HTTP.get(Webpacker.dev_server.host, file_path, Webpacker.dev_server.port)
-
-        begin
-          Tempfile.new(file_path).tap do |file|
-            file.binmode
-            file.write(asset)
-            file.rewind
-          end
-        rescue StandardError => e
-          Rails.logger.error "Error creating tempfile: #{e}"
-          raise
-        end
+        dev_server_asset(@asset_path)
+      elsif Webpacker.config.public_path.present?
```

Comment:
I wonder if we can get rid of this check.  I can't think of a scenario where an application using webpacker would _not_ have `Webpacker.config.public_path` configured. 🤔